code/game/machinery/doors/door.dm

```diff
@@ -327,7 +328,7 @@
 	update_freelook_sight()
 	if(safe)
 		CheckForMobs()
-	else
+	else if(can_crush)

```

Comment:
CheckForMobs shouldn't be called either

Reply:
Alright I just made it early return if it can't crush.